packages/fluent/scss/breadcrumb/_layout.scss

```diff
@@ -0,0 +1,132 @@
+
+@mixin kendo-breadcrumb--layout() {
+
+    // Breadcrumb
+    .k-breadcrumb {
+        margin-inline: var( --kendo-breadcrumb-margin-x, #{$kendo-breadcrumb-margin-x} );
+        margin-block: var( --kendo-breadcrumb-margin-y, #{$kendo-breadcrumb-margin-y} );
+        padding-inline: var( --kendo-breadcrumb-padding-x, #{$kendo-breadcrumb-padding-x} );
+        padding-block: var( --kendo-breadcrumb-padding-y, #{$kendo-breadcrumb-padding-y} );
+        border-width: var( --kendo-breadcrumb-border-width, #{$kendo-breadcrumb-border-width} );
+        border-style: solid;
+        box-sizing: border-box;
+        outline: 0;
+        font-family: var( --kendo-breadcrumb-font-family, #{$kendo-breadcrumb-font-family} );
+        font-size: var( --kendo-breadcrumb-font-size, #{$kendo-breadcrumb-font-size} );
+        line-height: var( --kendo-breadcrumb-line-height, #{$kendo-breadcrumb-line-height} );
+        display: flex;
+        flex-direction: row;
+        -webkit-touch-callout: none;
+        -webkit-tap-highlight-color: $rgba-transparent;
+
+        *,
+        *::before,
+        *::after {
+            box-sizing: border-box;
+        }
+    }
+
+
+    // Breadcrumb container
+    .k-breadcrumb-container,
+    .k-breadcrumb-root-item-container {
+        margin: 0;
+        padding: 0;
+        list-style: none;
+        display: flex;
+        flex-flow: row nowrap;
+        overflow: hidden;
+    }
+
+    .k-breadcrumb-container {
+
+        &.k-flex-wrap {
+            flex-wrap: wrap;
+        }
+    }
+    .k-breadcrumb-root-item-container {
+        align-items: flex-start;
+        flex-shrink: 0;
+    }
+
+
+    // Breadcrumb item
+    .k-breadcrumb-item {
+        vertical-align: middle;
+        display: inline-flex;
+        flex-direction: row;
+        align-items: center;
+        overflow: hidden;
+    }
+
+
+    // Breadcrumb root item
+    .k-breadcrumb-root-item {
+        flex-shrink: 0;
+    }
+
+
+    // Breadcrumb links
+    .k-breadcrumb-link,
+    .k-breadcrumb-root-link {
+        @include border-radius( var( --kendo-breadcrumb-border-radius, #{$kendo-breadcrumb-link-border-radius} ) );
+        padding-inline: var( --kendo-breadcrumb-link-padding-x, #{$kendo-breadcrumb-link-padding-x} );
+        padding-block: var( --kendo-breadcrumb-link-padding-y, #{$kendo-breadcrumb-link-padding-y} );
+        color: var( --kendo-breadcrumb-link-initial-text, #{$kendo-breadcrumb-link-initial-text} );
+        text-decoration: none;
+        white-space: nowrap;
+        outline: none;
+        cursor: pointer;
+        position: relative;
+        display: inline-flex;
+        align-items: center;
+        align-self: stretch;
+        overflow: hidden;
+        transition: var( --kendo-transition, #{$kendo-transition} );
+
+       &.k-selected {
+            font-weight: var( --kendo-breadcrumb-link-selected-font-weight, #{$kendo-breadcrumb-link-selected-font-weight} );
+       }
+
+       &:disabled,
```

Comment:
Should we move this in the theme file?

Reply:
I've placed the styles here, because they set the background and border to `none`, but I am okay with moving them if that is better in your opinion.